code/game/machinery/cryopod.dm

```diff
@@ -394,6 +394,7 @@
 				spawn(1) //This should ideally fire after the occupant is deleted.
 					if(!O) return
 					O.find_target()
+					O.owner.announce_objectives()
```

Comment:
This might work better after the if block below, so that it doesn't announce the objective if it's got a null target